src/actions/sources/loadSourceText.js

```diff
@@ -63,6 +65,7 @@ export function loadSourceText(source: SourceRecord) {
           [PROMISE]: loadSource(source, { sourceMaps, client })
         }: Action)
       );
+      telemetry.finish(loadSourceHistogram, source);
```

Comment:
I'm not sure about the telemetry api, does this support potential race conditions with timigh

```js    
telemetry.start(loadSourceHistogram, {id: 1});
telemetry.start(loadSourceHistogram, {id: 2});
telemetry.finish(loadSourceHistogram, {id: 2});
telemetry.finish(loadSourceHistogram, {id: 1});
```

also, do we want to handle timing when the load fails? Should we note that in the probe?

Reply:
As far as race conditions are concerned we are covered because we pass in source as an object and the timing event is then keyed on that source object:
`telemetry.finish(loadSourceHistogram, source);`

It is up to you whether you want to handle timing when the load fails... we could always move it to a finally block.